config_supported.go

```diff
@@ -2,7 +2,12 @@
 
 package wazero
 
-// NewRuntimeConfig returns NewRuntimeConfigCompiler
+// This file's implementation of the below function always returns compiler, but the doc for it should
```

Comment:
instead of copy/pasting stuff like this seems easier to solve this with an unexported function.

ex. move this to config.go
```
// NewRuntimeConfig returns a RuntimeConfig using the compiler if it is supported in this environment,
 // or the interpreter otherwise.
 func NewRuntimeConfig() RuntimeConfig {
  return newRuntimeConfig()
}
```

then you don't need to copy/paste these docs in the _supported and _unsupported files. instead, unexport NewRuntimeConfig to newRuntimeConfig

Reply:
Yup fixed

Reply:
thanks tons!